shell/edit/provisioning.cattle.io.cluster/rke2.vue

```diff
@@ -1808,16 +1575,6 @@ export default {
       this.syncChartValues(name);
     },
 
-    initQuestions(name) {
```

Comment:
comment only: good removal! i'm not going to admit it was me that introduced this and it was never used. nu uh.